logger/dapr_logger.go

```diff
@@ -4,6 +4,7 @@ import (
 	"os"
 	"time"
 
+	"github.com/natefinch/lumberjack"
```

Comment:
Should this be `gopkg.in/natefinch/lumberjack.v2`? That might clean up `go.mod` a bit.

Something to note: This package was last maintained in 2017.